test/test_utils/__init__.py

```diff
@@ -35,6 +35,8 @@
 DEFAULT_REGION = "us-west-2"
```

Comment:
We are using explicit regions declared here for parameterizing `"ec2_instance_type,region"` in the test.
However, the same is not used in the `region` declaration in conftest